packaging/suse/bundler-dumpdeps

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/ruby
+# vim: set sw=2 sts=2 et tw=80 :
+require 'bundler'
+
+def expand_pessimistic(version)
+  splitted = version.to_s.split('.')
+  sub_version = splitted.slice(0,(splitted.length-1)).join('.')
+  ":#{sub_version} >= #{version}"
+end
```

Comment:
I find this logic a bit weird, isn't `~> x.y.z` just meaning `>= x.y.z && < x.(y + 1)` ?

Reply:
I think expand_pessimistic will return ":x.y >= x.y.z". For example if you have "foo ~> 1.2.3" you will get "foo:1.2 >= 1.2.3"